tethys_apps/migrations/0004_auto_20230223_1731.py

```diff
@@ -0,0 +1,107 @@
+# Generated by Django 3.2.16 on 2023-02-23 17:31
```

Comment:
I would recommend deleting all of these migrations, rebasing this branch, setting up a new DB, and then re-create a single migration.

Reply:
Done